package.json

```diff
@@ -2,13 +2,14 @@
   "name": "redux-sentry",
   "version": "1.1.1",
   "description": "Middleware that logs all your store and actions on exception to Sentry with raven-js",
-  "main": "lib/middleware.js",
+  "main": "dist/middleware.js",
   "scripts": {
     "compile": "tsc -p ./",
     "watch": "tsc -watch -p ./",
     "commit": "./node_modules/.bin/git-cz",
     "version": " ./node_modules/.bin/conventional-changelog -i CHANGELOG.md -s && git add CHANGELOG.md",
-    "prepublish": "npm run compile"
+    "prepublish": "npm run compile",
```

Comment:
@Blackheart340, I think you want to also run `build` before `prepublish` script. Aren't you?